lapack.go

```diff
@@ -136,3 +136,23 @@ const (
 	InitZ   Comp = 'I'
 	UpdateZ Comp = 'V'
 )
+
+// EigVecSide specifies what eigenvectors will be computed.
+type EigVecSide byte
+
+// EigVecSide constants for Dtrevc3.
+const (
+	RightEigVec     EigVecSide = 'R' // Compute right eigenvectors only.
+	LeftEigVec      EigVecSide = 'L' // Compute left eigenvectors only.
+	RightLeftEigVec EigVecSide = 'B' // Compute both right and left eigenvectors.
+)
+
+// HowMany specifies which eigenvectors will be computed.
+type HowMany byte
+
+// HowMany constants for Dhseqr.
+const (
+	AllEigVec      HowMany = 'A' // Compute all right and/or left eigenvectors.
+	AllEigVecMulQ  HowMany = 'B' // Compute all right and/or left eigenvectors multiplied by an input matrix.
```

Comment:
At least the `Q` refers to the documentation for Dhseqr where `Q` denotes the matrix. I was also thinking about `AllEigVecUpdate` but I'm not sure if it's better. Lapack uses "backtransformed" but that's even worse.

I will revisit these constants again later. I'd like to use `EV` instead of `EigVec` but that's ambiguous with EigenValues.